commit a47bd7e3737276dc744f3aa76dcb55eaf27ee2a9
Author: Yang Bo <yangbo@linuxdeepin.com>
Date:   Sun Jan 3 19:55:57 2016 +0800

    UBUNTU: SAUCE: Revert "phy: vitesse make vsc824x_add_skew static"
    
    This is needed by p4080.
    
    This reverts commit baec126cf6a864e0191cf51ac1940f3c4c211617.
    
    Signed-off-by: Ben Collins <ben.c@servergy.com>
    
    Signed-off-by: Yang Bo <yangbo@linuxdeepin.com>

diff --git a/drivers/net/phy/vitesse.c b/drivers/net/phy/vitesse.c
index 76cad71..b45741f 100644
--- a/drivers/net/phy/vitesse.c
+++ b/drivers/net/phy/vitesse.c
@@ -74,7 +74,7 @@ MODULE_DESCRIPTION("Vitesse PHY driver");
 MODULE_AUTHOR("Kriston Carson");
 MODULE_LICENSE("GPL");
 
-static int vsc824x_add_skew(struct phy_device *phydev)
+int vsc824x_add_skew(struct phy_device *phydev)
 {
 	int err;
 	int extcon;
@@ -94,6 +94,7 @@ static int vsc824x_add_skew(struct phy_device *phydev)
 
 	return err;
 }
+EXPORT_SYMBOL(vsc824x_add_skew);
 
 static int vsc824x_config_init(struct phy_device *phydev)
 {
